server/src/main/java/com/google/android/attestation/AuthorizationList.java

```diff
@@ -393,8 +394,6 @@ public enum OperationPurpose {
 
   public abstract Optional<AttestationApplicationId> attestationApplicationId();
 
-  public abstract Optional<ByteString> attestationApplicationIdBytes();
```

Comment:
Yes, it is parsed into `attestationApplicationId`, which is what all google3 users are calling. Also if you really want the bytes for some reason, there is a method to marshal the `AttestationApplicationId` class back to DER: https://github.com/google/android-key-attestation/blob/ef45f71e08a774a568e733abf9e1921c86c84574/server/src/main/java/com/google/android/attestation/AttestationApplicationId.java#L115

Reply:
If I remember correctly, I was told that this method was left here because some implementations did not encode the attestation application ID correctly.